throttle/contract_tx_limiter_middleware.go

```diff
@@ -179,12 +237,19 @@ func NewContractTxLimiterMiddleware(cfg *ContractTxLimiterConfig,
 			}
 			txl.tierMap[contractTierID] = tierInfo
 		}
+		tierMapLoadDuration := time.Since(start)
+		lvs := []string{"method", "tierMap Load", "error", fmt.Sprint(err != nil)}
+		tierMapLoadLatency.With(lvs...).Observe(tierMapLoadDuration.Seconds())
 
 		if txl.isAccountLimitReached(contractAddr, state.Block().Height) {
 			return loomchain.TxHandlerResult{}, ErrTxLimitReached
 		}
-
+		start = time.Now()
 		txl.updateState(contractAddr, state.Block().Height)
+		contractStatsMapLoadDuration := time.Since(start)
+		lvs = []string{"method", "contract Stats Map Load", "error", fmt.Sprint(err != nil)}
+		contractStatsMapLoadLatency.With(lvs...).Observe(contractStatsMapLoadDuration.Seconds())
```

Comment:
This isn't going to be a very useful metric, it doesn't load anything, just accesses what's already in memory, remove it.

Reply:
Done.